php/libraries/UserPermissions.class.inc

```diff
@@ -163,6 +156,24 @@ class UserPermissions
         return true;
     }
 
+    /**
+     * Used to test whether a user has ANY of an array of specified permissions.
+     * Returns true on the first permission that the user has.
+     *
+     * @param array $permissions Array of permissions codes.
```

Comment:
probably better to be "string[]" than an untyped array.

Reply:
Didn't know you could do that! I'll make the change.